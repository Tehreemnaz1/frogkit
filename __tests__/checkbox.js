import { mount } from 'vue-test-utils'
import Vue from 'vue'
import Checkbox from '@/components/checkbox'

describe('Checkbox component', () => {

  it('should render component and match snapshot', () => {
    const wrapper = mount(Checkbox, { propsData: { name: 'checkbox-button'} })
    expect(wrapper.html()).toMatchSnapshot()
  })

  it('should render disabled checkbox', () => {
    const wrapper = mount(Checkbox, { propsData: { disabled: true } })
    expect(wrapper.html()).toMatchSnapshot()
  })

})
