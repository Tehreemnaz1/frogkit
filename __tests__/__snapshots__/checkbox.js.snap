// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox component should render component and match snapshot 1`] = `<label class="fk-checkbox"><input type="checkbox" name="checkbox-button"> </label>`;

exports[`Checkbox component should render disabled checkbox 1`] = `<label class="fk-checkbox fk-checkbox--disabled"><input type="checkbox" name="" disabled="disabled"> </label>`;
